pkg/controllers/provisioning/suite_test.go

```diff
@@ -857,26 +874,18 @@ var _ = Describe("Provisioning", func() {
 			)
 			ExpectProvisioned(ctx, env.Client, cluster, cloudProvider, prov, pod)
 			node := ExpectScheduled(ctx, env.Client, pod)
+
+			// We have multiple instance types that we can launch, one with 2Gi and one with 4Gi
+			// If we launch with 2Gi, this means the Daemon pod was respected
+			// If we launch with 4Gi, this means the Daemon pod was respected
 			allocatable := instanceTypeMap[node.Labels[corev1.LabelInstanceTypeStable]].Capacity
 			Expect(*allocatable.Cpu()).To(Equal(resource.MustParse("4")))
 			Expect(*allocatable.Memory()).To(Equal(resource.MustParse("4Gi")))
 		})
-		It("should account for daemonset spec affinity", func() {
+		It("should remove daemonset node hostname affinity when considering daemonset schedulability", func() {
+			// We make two NodePools: one that can't schedule the DaemonSet due to no matching
```

Comment:
It looks like this comment needs to be updated? There's only the one NodePool, and it is compatible with the DS.